addons/website/static/src/snippets/s_masonry_block/s_masonry_block.xml

```diff
@@ -0,0 +1,268 @@
+<?xml version="1.0" encoding="utf-8"?>
+<template id="masonry_template">
+    <t t-name="website.s_template_masonry.default">
+        <div class="container-fluid">
+            <div class="row">
+                <div class="col-lg-6 oe_img_bg text-center pb224 pt224" data-name="Block"
+                     style="background-image: url(/web/image/website.s_masonry_block_default_image_1);">
+                    <p><br/></p>
+                </div>
+                <div class="col-lg-6">
+                    <div class="row h-100">
+                        <div class="col-lg-6 pt24 pb8 text-center bg-200" data-name="Block">
+                            <h3>A great title</h3>
+                            <p>And a great subtitle</p>
+                        </div>
+                        <div class="col-lg-6 pt24 pb8 text-center o_cc o_cc3" data-name="Block">
+                            <h3>A great title</h3>
+                            <p>And a great subtitle</p>
+                        </div>
+                        <div class="col-lg-6 pt24 pb8 text-center o_cc o_cc4" data-name="Block">
+                            <h3>A great title</h3>
+                            <p>And a great subtitle</p>
+                        </div>
+                        <div class="col-lg-6 pt24 pb8 text-center bg-200" data-name="Block">
+                            <h3>A great title</h3>
+                            <p>And a great subtitle</p>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </div>
```

Comment:
In order to avoid duplicating the code in both `snippets.xml` and here for the default template, what you can do is the following:

- remove all the content of the snippet in `snippets.xml`
- in the start method of the snippet option, check if the block is empty, if it is, render the default template into the section and mark it as selected.